solutions/cf-1216C.mdx

```diff
@@ -0,0 +1,232 @@
+---
+id: cf-1216C
+source: CF 1216C
+title: D3C - White Sheet
+author: David Zhang
+---
+
+## Info/Tips
```

Comment:
wrap it in `<Info>`
just try that

Reply:
yeah and also make sure to set the title tag

see https://usaco.guide/general/working-mdx?lang=cpp#info-block

set it as "Info/Tips" or anything else if you come up with a better title

Reply:
*not necessary, but makes it look a lot better